Cargo.toml

```diff
@@ -77,39 +77,40 @@ strum_macros = "0.27.1"
+kona-mpt = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.2.0" }
+kona-derive = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.2.0", default-features = false }
+kona-driver = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.2.0" }
+kona-preimage = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.2.0", features = [
     "rkyv",
     "serde",
 ] }
-kona-executor = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.1.3" }
-kona-proof = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.1.3" }
+kona-executor = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.2.0" }
+kona-proof = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.2.0" }
 
-kona-client = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.1.3" }
-kona-host = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.1.3" }
-kona-providers-alloy = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.1.3" }
-kona-rpc = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.1.3", default-features = false }
-kona-protocol = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.1.3", default-features = false }
-kona-registry = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.1.3", default-features = false }
-kona-genesis = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.1.3", default-features = false }
+kona-client = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.2.0" }
+kona-host = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.2.0" }
+kona-providers-alloy = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.2.0" }
+kona-rpc = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.2.0", default-features = false }
+kona-protocol = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.2.0", default-features = false }
+kona-registry = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.2.0", default-features = false }
+kona-genesis = { git = "https://github.com/op-rs/kona", tag = "kona-client/v1.2.0", default-features = false }
 
 # hana
-hana-blobstream = { git = "https://github.com/celestiaorg/hana", tag = "v1.2.0-mocha" }
-hana-celestia = { git = "https://github.com/celestiaorg/hana", tag = "v1.2.0-mocha" }
-hana-host = { git = "https://github.com/celestiaorg/hana", tag = "v1.2.0-mocha" }
-hana-oracle = { git = "https://github.com/celestiaorg/hana", tag = "v1.2.0-mocha" }
+hana-blobstream = { git = "https://github.com/celestiaorg/hana", rev = "a027840b4c17462385816209a15ef206e1cd11c6" }
+hana-celestia = { git = "https://github.com/celestiaorg/hana", rev = "a027840b4c17462385816209a15ef206e1cd11c6" }
+hana-host = { git = "https://github.com/celestiaorg/hana", rev = "a027840b4c17462385816209a15ef206e1cd11c6" }
+hana-oracle = { git = "https://github.com/celestiaorg/hana", rev = "a027840b4c17462385816209a15ef206e1cd11c6" }
 
```

Comment:
Same note as hokulea. Please update this to the head revision after your PR merges.

Reply:
Do you think it should be blocking for merging this PR?

Reply:
https://github.com/succinctlabs/op-succinct/pull/668#discussion_r2479641142